### PR TITLE
Feature/taimoor test cases

### PR DESCRIPTION
**Pull request by taimoormalik97**
This pull request contains test cases.

### CODE REVIEW

File: spec/factories.rb [hunk 1]
```diff
@@ -0,0 +1,81 @@
+require 'faker'
+
+FactoryGirl.define do
+
+  factory :hospital, class: Hospital do
+    name { Faker::Name.unique.name }
+    address { Faker::Name.unique.name }
+    phone_number { Faker::PhoneNumber.cell_phone_in_e164 }
+    sub_domain { name.delete(' ').downcase }
+    #phone_number { Faker::Number.digit }
```
Comment (MirahSaqib) [inline spec/factories.rb]: remove comment

File: spec/models/admin_spec.rb [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      end
+
+      it 'should return true when different email is given in same sub-domain' do
+        expect(@admin.save).to eq true
+        @admin2.email = 'Email2@gmail.com'
+        expect(@admin2.save).to eq true
+      end
+    end
+  end
+end
```
Comment (MirahSaqib) [inline spec/models/admin_spec.rb:63]: Good Job! (y)

File: app/views/doctors/index.html.erb [hunk 3]
```diff
@@ -11,6 +11,38 @@
...
+            <th><%= t('doctor.label.registration_no') %></th>   
+            <th><%= t('doctor.label.speciality') %></th>   
+            <th><%= t('doctor.label.consultancy_fee') %></th>
+          </tr>   
+        </thead>   
+        <tbody>   
+        <% @doctors.each do |doctor| %>   
+          <tr>
+            <td>
+              <a href=<%= doctor_path(doctor) %> >
```
Comment (sulemanuzair) [inline app/views/doctors/index.html.erb]: use link_to tag

File: app/views/doctors/index.html.erb [hunk 4]
```diff
@@ -11,6 +11,38 @@
       </div>
     </div> 
     <div class="card-body">
+      <table  class="table picture_text">   
+        <thead >
```
Comment (sulemanuzair) [inline app/views/doctors/index.html.erb]: space

File: app/views/doctors/index.html.erb [hunk 5]
```diff
@@ -11,6 +11,38 @@
...
+            <th><%= t('doctor.label.consultancy_fee') %></th>
+          </tr>   
+        </thead>   
+        <tbody>   
+        <% @doctors.each do |doctor| %>   
+          <tr>
+            <td>
+              <a href=<%= doctor_path(doctor) %> >
+              <div class="set_image">
+                <% if doctor.profile_picture.present? %>
```
Comment (sulemanuzair) [inline app/views/doctors/index.html.erb]: make a helper function for this

File: config/application.rb [hunk 6]
```diff
@@ -11,6 +11,7 @@ module HospitalManagementSystem
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 6.0
+    config.action_dispatch.tld_length = 0
```
Comment (sulemanuzair) [inline config/application.rb:14]: why changing this ?

Comment (taimoormalik97) [reply config/application.rb:14]: because without it @request.host = "#{@hospital.sub_domain}.localhost:3000" is not working

**Pull request merged**